target/arduino/export.bat

```diff
@@ -0,0 +1,89 @@
+@ECHO off > nul
+:: Batch file to install BasicMAC from the git repository in to an
+:: existing Arduino installation by specifying the location of the 
+:: User library directory e.g.: "C:\Users\UserName\Documents\Arduino\libraries"
+
+:: \name Remko Welling (pe1mew@gmail.com)
+:: \date 8-7-2020
+:: \version 1.0
+
+:: \todo This script cannot work when whitespaces are in the directory path.
+:: As a workaround, export the library else where and move it in to the Arduino library directory.
+
+SET option=%1
+SET source_directory=%~dp0
+SET target_directory=%2
+
+2>NUL CALL :CASE_%option% 
+
+IF ERRORLEVEL 1 CALL :DEFAULT_CASE
+
+EXIT /B
+
+:CASE_-c
+:CASE_--create
+   IF EXIST %target_directory% (
+      GOTO :COPY
+   ) ELSE (
+      ECHO [ERROR] Destination directory %target_directory% does not exist. Please correct destination name.
+   )
+GOTO :EOF
+
+:CASE_-h
+:CASE_--help
+   ECHO Create an Arduino-compatible library in the given directory, overwriting existing files.
+   ECHO -h  --help   This information.
+   ECHO -c  --create ^<directory^> Create Arduino library at specified location
+GOTO :EOF
+
+:DEFAULT_CASE
+   ECHO Unknown command. Type: export.bat -h or --help for options.
+GOTO :EOF
+
+:COPY
+   :: Test if target directory has a terminating back-slash
+   IF NOT "%target_directory:~-1%"=="\" (
+      ECHO [ERROR] Target directory has no terminating "\" (back-slash^). Please correct target directory accordingly.
```

Comment:
Why is this needed? Below add an extra \ is added after the target directory already in some cases, can this not be always done? Or is it problematic for `COPY` to have a double `\\`?

Reply:
Double slashes are accepted in DOS/Windows. Solved by consistetly using slash in the script.